generate.sh

```diff
@@ -11,12 +11,12 @@ if [[ -z "${TT_REPO}" ]]; then
 	TT_REPO="https://github.com/tarantool/tarantool.git"
 fi
 
-TT_DIR=tarantool
+TT_DIR=tarantool-$TT_TAG
```

Comment:
I understand why this is done, but it is really unlikely that I will need different versions of tarantool when running the generator. So up to you, be free to ignore the suggestion.
```suggestion
TT_DIR=tarantool
```

Reply:
I would prefer to keep it. As it is useful for debugging, when you need keep both versions to compare them.

Reply:
Ok than.